src/main/java/com/kodilla/ecommercee/domain/orders/Item.java

```diff
@@ -17,7 +17,7 @@
 public class Item {
     @Id
     @GeneratedValue
-    @Column(name = "item_id", unique = true)
+    @Column(name = "id", unique = true)
     private Long id;
     @ManyToOne
     @JoinColumn(name = "order_id")
```

Comment:
i to tez nie powinno byc w tym commicie, tez jest ta zmaina w masterze naniesiona